utils/environment.python.sh

```diff
@@ -1,5 +1,5 @@
 # FIXME: fasm should be installed into the running Python environment.
-BASE_DIR=${FPGA_TOOL_PERF_DIR}
+BASE_DIR=$(pwd)
```

Comment:
This seems wrong as it sets it to the current working directory rather than a real path?

Reply:
now the path is stored in an env variable when sourcing env.sh